docs/source/recipes/faq-md.md

```diff
@@ -277,10 +277,7 @@ threshold.
 Does it support Elastic Cloud's "Cloud ID"?
 ==========
 
-Not supported.
-
-In addition, there is a reason why we cannot handle it at present.
-ElastAlert 2 uses elasticsearch-py 7.0.0, because Elastic Cloud cloud_id support is from elasticsearch-py 7.0.2. 
+Cloud ID is not currently supported.
```

Comment:
Several months ago, Seth Larson, [advised us](https://github.com/jertel/elastalert2/issues/92#issuecomment-928151918) that cloud ID is supported via the URL:

> Elastic Cloud is definitely supported by all 7.x versions of the client, the cloud_id is only a convenient way of specifying the Elasticsearch cluster you're connecting to. Specifying via a URL works just as well for Elastic Cloud.

Is there more work needed to support the URL method in ElastAlert 2?

Reply:
No extra work is required to support connecting to Elastic Cloud via the URL method, but extra work is required to support using Cloud ID to connect to Elastic Cloud clusters.

Elastic Cloud supports authentication via two methods, standard URL _or_ [Cloud ID](https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html). Configuring the connection to a cluster via Cloud ID is marginally more convenient, but isn't currently supported in ElastAlert.

The standard URL method of connecting works just fine.

Reply:
Thanks for clarifying. I think the FAQ answer should say something to that affect. Ex:

While Elastic Cloud is supported via the traditional URL connection method, connecting via Cloud ID is not currently supported.